lib/matplotlib/tests/test_image.py

```diff
@@ -1128,7 +1129,8 @@ def test_exact_vmin():
 @pytest.mark.network
 @pytest.mark.flaky
 def test_https_imread_smoketest():
-    v = mimage.imread('https://matplotlib.org/1.5.0/_static/logo2.png')
+    with _api.suppress_matplotlib_deprecation_warning():
+        v = mimage.imread('https://matplotlib.org/1.5.0/_static/logo2.png')
```

Comment:
Shouldn't you properly fix this?

Reply:
The point is that after the removal, reading this just won't involve matplotlib at all (it'll be `PIL.Image.open(urllib.request.urlopen(...))`, so the whole test will go away.

Reply:
oops sorry - I missed what the test was for....